### ISSUE TITLE
Android testImplementation() error after fresh install

### ISSUE THREAD

**Issue by fatihsolhan** [developer239/redux-react-native-wix-navigation-v2-with-auth#63]
Hi,
After fresh installing I got an error on android build.
`Could not find method testImplementation() for arguments [org.khronos:opengl-api:gl1.1-android-2.1_r1] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.`

`yarn ios` works perfect.
But on `yarn android` I got error

Could you please help me about this? Thank you

**Comment by developer239** [comment]
Hello, @fatihsolhan thanks for using this boilerplate. Honestly, I didn't run it for a while even though I am trying to keep it alive by using automatic dependency updates and by manual changes that I once every couple of months. 🙂 

Right now I am playing around with https://github.com/developer239/create-opinionated-react-app

Hopefully, I will regenerate this boilerplate this weekend or sometimes in next couple of weeks.

**Comment by developer239** [comment]
@fatihsolhan 🎉I just updated the codebase. iOS and Android should work again with latest dependencies. I also added detox and one minimal test just to make sure that the CI can build the application and then run it and show the home screen.

Greenkeeper was replaced with Dependabot 🤖 so dependency updates shouldn't break the application now.

I want to fix the eslint issues tomorrow. I will see how well it is going to go.
